src/components/AboutMe/AboutMe.tsx

```diff
@@ -8,22 +8,26 @@ function AboutMe() {
     return (
         <div className={styles.aboutMe}>
             <div className="container">
-                <img src={imgSource} alt="" className={styles.img} />
+                <img src={imgSource} alt="Beautiful stairs" className={styles.img} />
                 <div className={styles.content}>
                     <h1 className={styles.title}>About me</h1>
                     <p className={styles.text}>
                         Hi! My name is Anton and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like
                         React, HTML, CSS, JavaScript and Git version control system. <br /> <br />
-                        This page was developed during the course &apos;Intro to React&apos; from Masters Academy in 2024. <br /> <br />
+                        This page was developed during the course&#32;
+                        <a href="https://www.mastersacademy.education/" target="_black" className={styles.link}>
```

Comment:
Considering the repeated use of `<a>` tags with identical functionality and styling throughout the application, it would be beneficial to encapsulate this pattern into a reusable component. 
This approach will centralize common logic and styles, facilitating easier updates and ensuring consistency across the application. Please create a helper component for the `<a>` tags to streamline code maintenance and improve readability.

```tsx
<Link href target ...etc></Link>
```
component should be able accept any props which native `a` tag can accept

Hint: Go to definiton files of a tag you can do it by `ctrl + click` or `cmd + click` on MacOs
https://mortenbarklund.com/blog/react-typescript-props-spread/
![Screenshot 2024-04-19 at 18 10 36](https://github.com/Allocene/fe-react-2024/assets/55930010/e9051de3-4049-4ac1-b776-ab02dbe7ffc2)

Reply:
Might be fixed.